src/config/env.ts

```
@@ -61,7 +61,9 @@ envCheck(
 	"MICROS_AWS_REGION",
 	"GLOBAL_HASH_SECRET",
 	"CRYPTOR_URL",
-	"CRYPTOR_SIDECAR_CLIENT_IDENTIFICATION_CHALLENGE"
+	"CRYPTOR_SIDECAR_CLIENT_IDENTIFICATION_CHALLENGE",
+	"MICROS_PLATFORM_STATSD_HOST",
```

Make sure it fails fast